CMakeLists.txt

```diff
@@ -5,6 +5,14 @@ if (NOT DEFINED ENV{TTMLIR_ENV_ACTIVATED})
   message(FATAL_ERROR "tt-mlir environment not activated. Please run 'source env/activate'.")
 endif()
 
+option(PERF_MODE "Enable performance mode" OFF)
```

Comment:
Let's keep this line here, but rename the variable to be `TT_RUNTIME_ENABLE_PERF_TRACE`.